sendgrid/sendgrid.py

```diff
@@ -45,4 +45,4 @@ def api_key(self):
 
     @api_key.setter
     def api_key(self, value):
-        self._apikey = value
```

Comment:
Something strange seems to have happened with this PR. Now, the only change I see is a return key. Can you verify?